src/instructlab/rag/convert.py

```diff
@@ -175,7 +177,13 @@ def _initialize_docling():
         artifacts_path=docling_model_path,
         do_ocr=False,
     )
-    ocr_options = _resolve_ocr_options()
+
+    # deactivate MPS acceleration on Github CI
+    if os.getenv("CI") and sys.platform == "darwin":
```

Comment:
It could be cleaner if we'd have a variable with a very explicit name here, instead of `CI`. E.g. `INSTRUCTLAB_DISABLE_GPU_ACCELERATION`, and then we don't need to check for platform since we'll set the variable only for macos runs.

Reply:
updated